http/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20210225_add_zone.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="leonardo" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <changeSet logicalFilePath="leonardo" author="willyn" id="add_zone">
+        <addColumn tableName="RUNTIME_CONFIG">
+            <column name="zone" type="varchar(254)">
+                <constraints nullable="false"/>
```

Comment:
If adding a new non-nullable field to an existing table it should be coupled with a data migration:
```
1. add nullable column
2. `UPDATE RUNTIME_CONFIG set zone = 'us-central1-a'`
3. Make column non-nullable